src/driver/src/DefaultDriver.cpp

```diff
@@ -240,10 +226,16 @@ genny::driver::DefaultDriver::ProgramOptions::ProgramOptions(int argc, char** ar
     this->isHelp = vm.count("help") >= 1;
     this->shouldListActors = vm.count("list-actors") >= 1;
     this->isDryRun = vm.count("dry-run") >= 1;
-    this->metricsFormat = vm["metrics-format"].as<std::string>();
-    this->metricsOutputFileName = normalizeOutputFile(vm["metrics-output-file"].as<std::string>());
+    this->metricsFormats = vm["metrics-format"].as<std::vector<std::string>>();
+    this->metricsOutputFileNames = vm["metrics-output-file"].as<std::vector<std::string>>();
     this->mongoUri = vm["mongo-uri"].as<std::string>();
 
+    if (this->metricsFormats.size() != this->metricsOutputFileNames.size()) {
+        throw std::invalid_argument(
+            "the 'metrics-format' and 'metrics-output-file' options must have the same number of "
+            "arguments");
+    }
```

Comment:
this seems confusing...I think I get why you went with two vectors now though...
would it instead be easier to allow eg..

```
genny -m csv output.csv -m csv2 output2.csv
```

i.e. where `-m` or (`--metrics-format`) takes two options first being type and second being output filename?